### ISSUE TITLE
Signup Page Satellite UI: Can't signup with tab control

### ISSUE THREAD

**Issue by nadimhq** [storj/storj#5163]
**Description**
On signup page, one can use the tab button and keyboard only inputs to select satellite, enter full name, email address, password and retyped password. However, one cannot click on the checkbox for "I agree to the Terms of Service and Privacy Policy", thus making it impossible to sign up with only tab control. Also one cannot signup with a business account using tab control as well, since it's impossible to select the business option with tab control and even even if that wasn't the case we would still face the same situation in that the checkboxes for a business account are also impossible to select using tab control

**Step to Reproduce**
1. Open signup page
2. Use tab control to select desired region, input name, email and password into their respective fields and signup with new account

**Results**
As stated before, **expected** to be able to create account regardless of type (business, personal) or region, but **actually** it was impossible to create a personal account since tabbing wouldn't select the required checkbox- "I agree to the Terms of Service and Privacy Policy. It was also impossible to create a business account using tab only as there was no way to select the business account in the first place with tab control and even if it was possible we would still face the same situation with being unable to select the required checkbox

https://user-images.githubusercontent.com/82845192/189662394-7cd7d95b-b135-4fd9-aad6-ef3fc3444d20.mp4

AC:
* You can sign up using keyboard navigation
* Accessibility tools show no issues for the signup page (e.g. we may need to change screen reader tags) - [example tool](https://web.dev/measure/), but there are lots of others, and browser extensions

**Comment by mobyvb** [comment]
see https://github.com/storj/storj/commit/d3805761a20ae1a58eca19cf76eaff7fb3b30245

**Comment by mobyvb** [comment]
1.70 https://github.com/storj/storj/commit/99f4b2a26e6c96
